curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-json-with-the-javascript-fetch-method.english.md

```diff
@@ -97,7 +97,7 @@ tests:
   }
 </style>
 <h1>Cat Photo Finder</h1>
-<p id="message">
+<p id="message box">
```

Comment:
This challenge has a id of message, you added `box` as part of the id. It needs to be a class. This is the only challenge where `message` is the id and not the class. For now, I would just add `box` as a class, but we may want to go through and change this to align with the rest of the challenges. Or, if you want to take a look and maybe rewrite a little, go for it.

Reply:
Yeah, You`re right. I'll fix it and upload it again

Reply:
This file was removed from the PR